youtube_dl/extractor/bellesa.py

```diff
@@ -0,0 +1,109 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import json
+
+from ..utils import ExtractorError, clean_html, try_get
+from .common import InfoExtractor
+
+
+class BellesaIE(InfoExtractor):
+    _VALID_URL = r'''https?://www\.bellesa\.co/videos/(?P<id>[0-9]+)/'''
+    _TESTS = [{
+        'url': 'https://www.bellesa.co/videos/2189/my-first-time-kissing-women',
+        'md5': '543b002f4d1e5b4a0dde243f4a43fd28',
+        'info_dict': {
+            'id': '2189',
+            'ext': 'mp4',
+            'title': 'My First Time Kissing Women',
+            'thumbnail': 'https://c.bellesa.co/dkvdbifey/image/upload/v1599024046/video_upload/2189cover.jpg',
+            'description': 'Jenna opens up about her troubles navigating a long-distance relationship. Her girlfriends ask if she’d ever be open to cheating on him but she says she would never, they’ve been together since college. Lena and Carter sit on either side of her and remind her it isn’t cheating if it’s with a girl…they start kissing and rubbing her chest, going down on her together. This girl on girl scene is so sensual, and the nerves on Jenna are real — exploring your sexuality takes courage, but there’s a lot of good vibes in this threesome.',
```

Comment:
With a long description, use MD5:
```suggestion
            'description': 'md5:69eea8a4ee31d42d6fd6302ad9e09ab2',
```